scripts/pylib/twister/twisterlib/environment.py

```diff
@@ -840,6 +840,15 @@ def add_parse_arguments(parser = None) -> argparse.ArgumentParser:
         NOTE: west-flash must be enabled to use this option.
         """
     )
+    parser.add_argument(
+        "--flash-command",
+        help="""Uses a custom flash command instead of ninja or make to flash
```

Comment:
Did you mean?
```suggestion
        help="""Uses a custom flash command instead of west to flash
```
Context:
- #92406

Reply:
At least @tejlmand seems to care; in #92406 (which this supersedes - so it was IMHO worth linking them to each other) he just said `ninja flash` should die. So it did not seem like a good idea to mention that some `--help` text.

> dropped that whole "instead of bit"

Great!